### ISSUE TITLE
Uplink configuragtion 

### ISSUE THREAD

**Issue by Eyal1980** [thingsboard/thingsboard#11456]
**Component**

 * Installation

**Description**
I trying to make an integration for a physical device but there is a problem.
The integration is working and get the data from my physical device, but the uplink doesn't get it. attached screenshots photos.
How can i fix it?
![device](https://github.com/user-attachments/assets/3b85848f-78a5-4083-9b4f-8cda3d613bb3)
![integration](https://github.com/user-attachments/assets/4ed9fe62-188c-4bef-a93f-a5dd49610fb9)
![uplink json](https://github.com/user-attachments/assets/7719628a-1847-49fd-928d-5798ae4df99f)
![uplink in](https://github.com/user-attachments/assets/6f889527-6cc0-43bc-b21d-b2d67acfcfbf)
![uplink out](https://github.com/user-attachments/assets/a65b7b4f-a188-431d-9c46-0019fd82a633)

**Environment**

 * OS: Windows 11 Pro 23H2
 * ThingsBoard: Cloud
 * Browser: Chrome 127.0.6533.120

___________________________________________________________
**Disclaimer** 

We appreciate your contribution whether it is a bug report, feature request, or pull request with improvement (hopefully). Please comply with the [Community ethics policy](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies), and do not expect us to answer your requests immediately. Also, do not treat *GitHub issues* as a support channel.

**Comment by mykhailokornieiev** [comment]
Hi. This is related to an incorrect uplink converter script. 
It must be something like this:
![image](https://github.com/user-attachments/assets/816e8f2e-c2d6-4728-abda-aece69d27a88)
```
/** Decoder **/

// decode payload to string
var payloadStr = decodeToString(payload);
var deviceName =  "MyDevice123";
// decode payload to JSON
var deviceType = 'default';
var date = data.time;

// Result object with device attributes/telemetry data
var result = {
    deviceName: deviceName,
    deviceType: deviceType,
    telemetry: {
        PulseName: data.PulseName,
        PulseCount: data.PulseCount,
        FlowRate: data.FlowRate
    }
};

/** Helper functions 'decodeToString' and 'decodeToJson' are already built-in **/

return result;
```

**Comment by mykhailokornieiev** [comment]
Closed due to inactivity

**Comment by Eyal1980** [comment]
Thank you 🙏🏽 ‫ב-28 באוג׳ 2024, בשעה 17:02, ‏‏Mykhailo Kornieiev ***@***.***> כתב/ה:‬﻿
Closed #11456 as completed.
